tutorials/applications/QAOA and QAA to solve a MIS problem.ipynb

```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>Why these values in particular? These are questions people frequently pose, it would be best that there was some reasoning behind them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>